### PR TITLE
Add authentication in google calendar

### PR DESCRIPTION
**Pull request by kozlovpn**
Add authentication, show calendar events

### CODE REVIEW

File: shellper/main.py [hunk 1]
```diff
@@ -16,8 +17,12 @@ def main():
         config = yaml.load(yaml_file)
 
     validation.validate(config)
+    del sys.argv[-1]
 
     google = base.Base()
+
+    google.get_event_list()
```
Comment (ivanovmi) [inline shellper/main.py]: why we need it?

File: shellper/base.py [hunk 2]
```diff
@@ -1,4 +1,16 @@
+import argparse
```
Comment (esikachev) [inline shellper/base.py]: non-alphabetic order

File: shellper/main.py [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import argparse
 import yaml
+import sys
```
Comment (esikachev) [inline shellper/main.py]: ditto

File: shellper/tests/unit/test_base.py [hunk 4]
```diff
@@ -1,10 +1,29 @@
+import sys
+import mock
```
Comment (esikachev) [inline shellper/tests/unit/test_base.py]: need correct order

File: shellper/main.py (hunk 1 above)
Comment (esikachev) [inline shellper/main.py]: @ivanovmi this is call Pasha's method

Comment (ivanovmi) [inline shellper/main.py]: ok, i'm understand

**Comment by ivanovmi** [comment]
+1

**Comment by esikachev** [comment]
thak you

**Pull request merged**